src/blockfilter.h

```diff
@@ -13,6 +13,7 @@
 
 #include <primitives/block.h>
 #include <serialize.h>
+#include <streams.h>
```

Comment:
I don't really like moving these functions to the public interface of `blockfilter` and adding the `stream.h` dependency here. Maybe move this to a new header that is only included in the two places it's needed?

Reply:
Good point. Now moved to `golombrice.h`. Please re-review :)